src/main/java/com/revinate/sendgrid/model/Whitelabel.java

```diff
@@ -27,6 +29,22 @@ public Whitelabel(String domain, String subdomain) {
         this.subdomain = subdomain;
     }
 
+    public String getIp() {
+        return ip;
+    }
+
+    public void setIp(String ip) {
+        this.ip = ip;
+    }
+
+    public DnsRecord getaRecord() {
+        return aRecord;
+    }
+
+    public void setaRecord(DnsRecord aRecord) {
```

Comment:
The "a" should be capitalized. So the getter should be "getARecord" and the setter should be "setARecord".

Reply:
@alexleigh - The Json field is 'a_record'. Should the 'a' in a_record still be capitalized?